docs/pages/server-access/guides/ec2-discovery.mdx

```diff
@@ -21,7 +21,42 @@ install Teleport manually.)
 
 (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/5. Define an IAM policy
+## Step 1/6. Create an EC2 invite token
+
+When discovering EC2 instances, Teleport makes use of IAM invite tokens for
+authenticating joining Nodes.
+
+Create a file called `token.yaml`:
+
+```yaml
+# token.yaml
+kind: token
+version: v2
+metadata:
+  # the token name is not a secret because instances must prove that they are
+  # running in your AWS account to use this token
+  name: aws-discovery-iam-token
+  # set a long expiry time, as the default for tokens is only 30 minutes
+  expires: "3000-01-01T00:00:00Z"
```

Comment:
I would also ask @lxea to address this since this came from their issue.

Reply:
Sounds good to me, no @ptgott ?

Reply:
Works for me! I had this question because I didn't understand how EC2 discovery works, which touches on earlier comment—it would be really nice to have an architectural overview of EC2 discovery at the beginning of the guide, because otherwise the way this feature works is pretty murky.